test/evergreen.yml

```diff
@@ -1965,3 +2039,5 @@ buildvariants:
   tasks:
   - name: compile
   - name: unit-test
+  - name: recovery-stress-test-without-lz4 # Evergreen Z-series distro doesn't support lz4
```

Comment:
Will you be creating another follow-on ticket to replace this task once the zSeries Evergreen distro get lz4 installed?

Reply:
I am not sure on this one. Should we wait for zSeries distro fix or should we go ahead like this and create a Jira ticket in to remove this special function once distro fix is in place?